corehq/apps/domain/templates/domain/bootstrap5/confirm_plan.html

```diff
@@ -26,6 +26,7 @@
     >
       {% csrf_token %}
       <input type="hidden" value="{{ plan.edition }}" name="plan_edition" />
+      <input type="hidden" value="{{ is_annual_plan }}" name="is_annual_plan" />
```

Comment:
might be good to use `BooleanField` in the form and crispy forms if possible...would reduce overall html boilerplate and would have the added benefit of not needing the `lower() == 'true'` check below. not blocking, but should be a followup item.

Reply:
ed7bcc3